docs/commands/rhoas_kafka_create.md

```diff
@@ -31,7 +31,7 @@ $ rhoas kafka create -o yaml
 
 ```
       --name string       Unique name of the Kafka instance
-  -o, --output string     Specify the output format. Choose from: "json", "yaml", "yml" or "none"
+  -o, --output string     Specify the output format. Choose from: "json", "none", "yaml", "yml" (default "json")
```

Comment:
Do you edit those manually? They do not seem to be consisted across the board.

All list commands should have list and none extra
All other commands should have none as extra option

Reply:
Hi, I actually manually edited all to reflect none in my previous commit, but in this one, I used the generate doc option and it skipped none for a couple of these files, and when i checked each manually, from a logical standpoint it made sense to omit the none option, for commands like "describe"
I wanted to get your input on this before reverting the auto generated docs to reflect my own change.

Reply:
This one does require none, i will revert it to the previous version.

Reply:
Let's no edit any of this files. They are generated.